src/ReleaseHistory.md

```diff
@@ -1,9 +1,12 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## **v3.1.0** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/3.1.0) | [Driver](https://www.nuget.org/packages/Sarif.Driver/3.1.0) | [Converters](https://www.nuget.org/packages/Sarif.Converters/3.1.0) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/3.1.0) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/3.1.0) 
+
+* DEPENDENCY BREAKING: SARIF.SDK now requires `System.Collections.Immutable` 1.5.0. [#2504](https://github.com/microsoft/sarif-sdk/pull/2533)
+
 ## **v3.0.0** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/3.0.0) | [Driver](https://www.nuget.org/packages/Sarif.Driver/3.0.0) | [Converters](https://www.nuget.org/packages/Sarif.Converters/3.0.0) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/3.0.0) | [Multitool Library](https://www.nuget.org/packages/Sarif.Multitool.Library/3.0.0)
 * BUGFIX: Loosen Newtonsoft.JSON minimum version requirement to 6.0.8 (for .NET framework) or 9.0.1 (for all other compilations) for Sarif.Sdk. Sarif.Converts requires 8.0.1, minimally, for .NET framework compilations.
 * BUGFIX: Broaden set of supported .NET frameworks for compatibility reasons. Sarif.Sdk, Sarif.Driver and Sarif.WorkItems requires net461.
-* BUGFIX: Set default stack limit in Newtonsoft.JSON utilization (if `JsonConvert.Defaults` is not already configured) to address GitHub advisory [GHSA-5crp-9r3c-p9vr](https://github.com/advisories/GHSA-5crp-9r3c-p9vr).
```

Comment:
>BUGFIX: Set default stack limit in Newtonsoft.JSON utilization [](http://example.com/codeflow?start=2&length=62)

Removing this because we are not setting any limit anymore.

Reply:
We're no longer setting a limit? Which PR was this removed in and why?

Reply:
This got removed in this:
[https://github.com/microsoft/sarif-sdk/pull/2527](https://github.com/microsoft/sarif-sdk/pull/2527)

Reply:
We discussed today in the IcMs meeting about this. The point is: the consume will need to fix and the sarif.sdk, since its the lower level, it cannot enforce the fix on anyone.

Reply:
Got it. Thanks for the context.